src/sentry/integrations/slack/tasks/__init__.py

```diff
@@ -1,3 +1,3 @@
-from sentry.integrations.slack.tasks.send_notifications_on_activity import (  # noqa
-    send_activity_notifications_to_slack_threads,
-)
+from .send_notifications_on_activity import send_activity_notifications_to_slack_threads
+
+__all__ = ("send_activity_notifications_to_slack_threads",)
```

Comment:
oh i did see this in places but i didn't know it was key 😬

Reply:
I always forget too, it's the small things: https://stackoverflow.com/questions/44834/what-does-all-mean-in-python

> It is a list of strings defining what symbols in a module will be exported when from <module> import * is used on the module.